ffmpeg-examples/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+
+include_directories(${CMAKE_SOURCE_DIR})
+add_executable(sherpa-ncnn-ffmpeg sherpa-ncnn-ffmpeg.cc)
+target_link_libraries(sherpa-ncnn-ffmpeg pthread m)
+
+# Link libraries from sherpa-ncnn.
+target_link_libraries(sherpa-ncnn-ffmpeg sherpa-ncnn-c-api sherpa-ncnn-core kaldi-native-fbank-core ncnn)
+if(BUILD_SHARED_LIBS)
+    target_link_libraries(sherpa-ncnn-ffmpeg "portaudio")
+else()
+    target_link_libraries(sherpa-ncnn-ffmpeg "portaudio_static")
+endif()
```

Comment:
```suggestion
target_link_libraries(sherpa-ncnn-ffmpeg sherpa-ncnn-c-api)
```

Reply:
Fixed, cmake will automatically link the depended libraries such as portaudio